CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -13,6 +13,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 - Update [the compaction log when bootstrap](https://github.com/etcd-io/etcd/pull/17830).
 - Fix [Revision decreasing after panic during compaction](https://github.com/etcd-io/etcd/pull/17865)
 - Add [`etcd --experimental-stop-grpc-service-on-defrag`](https://github.com/etcd-io/etcd/pull/17914) to enable client failover on defrag.
+- Add [support for AllowedCN and AllowedHostname through config file](https://github.com/etcd-io/etcd/pull/18063)
```

Comment:
Nit: Prefer code formatting for field names.

```suggestion
- Add [support for `AllowedCN` and `AllowedHostname` through config file](https://github.com/etcd-io/etcd/pull/18063)
```

Reply:
Sure, let me fix it. Thanks!